test/unit/models/organisation_test.rb

```diff
@@ -995,7 +995,7 @@ class OrganisationTest < ActiveSupport::TestCase
     )
   end
 
-  test "#save triggers an update of the organisations index page links hash" do
+  test "#save triggers an update of the organisations index page links" do
```

Comment:
Can probably remove ` links` from the end of this

Reply:
Done